atacama.ipynb

```diff
@@ -0,0 +1,806 @@
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 161032441 (154M) [binary/octet-stream]\n",
+      "Saving to: ‘/home/user/Documents/qiime2-atacama-tutorial//emp-paired-end-sequences/reverse.fastq.gz’\n",
+      "\n",
+      "      /home/user/Do   0%[                    ] 255.00K   308KB/s               ^C\n"
+     ]
+    }
+   ],
+   "source": [
+    "!wget -O $workdir/\"emp-paired-end-sequences/reverse.fastq.gz\" \\\n",
+    "  \"https://data.qiime2.org/2020.2/tutorials/atacama-soils/10p/reverse.fastq.gz\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 74,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "--2020-06-03 20:54:22--  https://data.qiime2.org/2020.2/tutorials/atacama-soils/10p/barcodes.fastq.gz\n",
+      "Resolving data.qiime2.org (data.qiime2.org)... 52.35.38.247\n",
+      "Connecting to data.qiime2.org (data.qiime2.org)|52.35.38.247|:443... connected.\n",
+      "HTTP request sent, awaiting response... 302 FOUND\n",
+      "Location: https://s3-us-west-2.amazonaws.com/qiime2-data/2020.2/tutorials/atacama-soils/10p/barcodes.fastq.gz [following]\n",
+      "--2020-06-03 20:54:23--  https://s3-us-west-2.amazonaws.com/qiime2-data/2020.2/tutorials/atacama-soils/10p/barcodes.fastq.gz\n",
+      "Resolving s3-us-west-2.amazonaws.com (s3-us-west-2.amazonaws.com)... 52.218.184.152\n",
+      "Connecting to s3-us-west-2.amazonaws.com (s3-us-west-2.amazonaws.com)|52.218.184.152|:443... connected.\n",
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 19976093 (19M) [binary/octet-stream]\n",
+      "Saving to: ‘/home/user/Documents/qiime2-atacama-tutorial//emp-paired-end-sequences/barcodes.fastq.gz’\n",
+      "\n",
+      "        /home/user/   0%[                    ]  58.64K   130KB/s               ^C\n"
+     ]
+    }
+   ],
+   "source": [
+    "!wget -O $workdir/\"emp-paired-end-sequences/barcodes.fastq.gz\" \\\n",
+    "  \"https://data.qiime2.org/2020.2/tutorials/atacama-soils/10p/barcodes.fastq.gz\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Importing data as a qiime2 artifact\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "All data that is used as input to QIIME 2 is in form of QIIME 2 artifacts, which contain information about the type of data and the source of the data. So, the first thing we need to do is import these sequence data files into a QIIME 2 artifact.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "paired_end_sequences = qiime2.Artifact.import_data('EMPPairedEndSequences', workdir+'/emp-paired-end-sequences/')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Paired-end read analysis commands"
```

Comment:
This heading is a bit higher in the original tutorial. Although it is not misleading.